### ISSUE TITLE
Tech Debt: "Join a case" page should call case summary API

### ISSUE THREAD

**Issue by subhashini-egov** [pucardotorg/dristi#1402]
**Describe the bug**

When advocates or respondents try to join a case, they should be able to view some basic, public details of the case. Only after successfully joining a case should they be able to see the full details of a case.

**To Reproduce**
1. Login as a respondent
2. Click on 'Join a Case'
3. Enter the details and access code
4. Open the network tab
5. All details of a case are openly visible in the payload.

**Expected behavior**
Only public fields should be visible in the network tab.

**Additional context**
Related to the case summary API issue that has been filed.

**Comment by atulgupta2024** [comment]
API has been implemented as per ticket #1923 , but join a case implementation is still not using it. Need to fix that. 

@suresh12 request to please help update the correct status here.

**Comment by rajeshcherukumalli** [comment]
Closing the ticket with new ticket reference https://github.com/pucardotorg/dristi/issues/3037

**Comment by tivre-gupta1002** [comment]
Tested in SOL-DEV environment, working as expected.
Jam video link for reference: https://jam.dev/c/842b184d-f54e-4f92-90f8-fda751d8c193
cc: @rajeshcherukumalli

**Comment by tivre-beehyv** [comment]
Tested in UAT environment, working as expected.
Jam video link for reference: https://jam.dev/c/e4df985e-b19a-4734-962e-495213efdc4a
Hi @harshgupta-pucar, please verify once.
cc: @rajeshcherukumalli

**Comment by tivre-gupta1002** [comment]
Tested in UAT environment, working as expected.
Jam video link for reference: https://jam.dev/c/cd4021e2-e668-43bd-b684-ac47d70064fc
Hi @harshgupta-pucar, please verify once.
cc: @rajeshcherukumalli

**Comment by atulgupta2024** [comment]
closing
